MobileBuy/buy/src/androidTest/java/com/shopify/buy/service/BuyTest.java

```diff
@@ -188,7 +186,7 @@ public void failure(RetrofitError error) {
     @Test
     public void testWithoutShop() {
         try {
-            BuyClientFactory.getBuyClient("", getApiKey(), getAppId(), data.getApplicationName());
+            new BuyClientBuilder().shopDomain("").build();
```

Comment:
This works this way?  it seems like it would be order dependant on the checking, we should probably add a non-empty apiKey, appId